### DISCUSSION TITLE
question about core package

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [fastify/fastify#3890] [Q&A] [answered]
I noticed almost all fastify plugins changed package names under new brand name `@fastify/...` as fastify is approaching to the new major release. What about the main/core package itself? will it be the same or `@fastify/core` (or something like this)? For consistency, it would be nice to have `@fastify/core` instead of `fastify`

**Comment by jsumners** [comment, marked as answer]
Please read the pinned issues.
